packages/common/constants/environments.ts

```diff
@@ -0,0 +1,7 @@
+export const ENVIRONMENTS = {
+  DEVELOPMENT: 'development',
+  TESTING: 'testing',
+  PRODUCTION: 'production',
+};
+
+export const isTesting = () => process.env.NODE_ENV === ENVIRONMENTS.TESTING;
```

Comment:
In another PR, you are adding `isProduction`. Shall we also have `isDevelopment` so the list is complete?

Reply:
So you are suggesting adding `isDevelopment` function or you want to use `isDevelopment` instead of `isTesting` in this case. I am not quite sure what you mean.

Reply:
I will solve this in #1188. Adding it here will cause a bigger conflict.